presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -94,6 +99,12 @@ public void run()
         @Option(name = "--reports-dir", title = "reports dir", description = "location of the reports directory")
         public String reportsDir;
 
+        @Option(name = "--startup-timeout", title = "startup timeout", description = "environment startup timeout")
+        public Duration startupTimeout = Duration.valueOf("5m");
```

Comment:
5m is too little and env up should have the same timeout (currently env up has 10m and this has 5m).

but, Idk why we need a startup timeout now, that we didn't before. Did you experience startup freeze?
how often does it happen?

Reply:
5 min is enough for every environment that we use. I agree that test run and env up should have the same timeout. I'll open followup PR

Reply:
it's not, if you factor in image downloads.
especially that this can be run on GHA, but also locally.
10 or even 15m would feel much safer

Reply:
in fact, i'd rather not have a timeout on individual attempt, until this is proven necessary.
i don't remember seeing a startup hang